src/main/resources/data/coww/tweakers/test_cauldron_tweaker.js

```diff
@@ -0,0 +1,10 @@
+var CauldronRecipeTweaker = libcd.require("mods.coww.libcd.CauldronRecipeTweaker");
+CauldronRecipeTweaker.addCauldronRecipe(
```

Comment:
Definitely add an [mcmeta](https://github.com/CottonMC/LibCD/blob/1.15/src/main/resources/data/libcd/tweakers/tweaker_test.js.mcmeta) for this.